src/Illuminate/Database/Eloquent/ModelNotFoundException.php

```diff
@@ -35,11 +35,9 @@ public function setModel($model, $ids = [])
 
         $this->message = "No query results for model [{$model}]";
 
-        if (count($this->ids) > 0) {
-            $this->message .= ' '.implode(', ', $this->ids);
-        } else {
-            $this->message .= '.';
-        }
+        $this->message = count($this->ids) > 0
```

Comment:
How is this any better than the IFs? I find `if(){ ... }else{ ... }` much clearer.

Reply:
This shortens make a bit confuse. Teneray not always good.